client/asset/eth/eth.go

```diff
@@ -84,6 +84,10 @@ const (
 	// coinIDTakerFoundMakerRedemption is a prefix to identify one of CoinID formats,
 	// see DecodeCoinID func for details.
 	coinIDTakerFoundMakerRedemption = "TakerFoundMakerRedemption:"
+
+	// maxTxFeeGwei is the default max amout of eth that can be used in one
+	// transaction.
+	maxTxFeeGwei = 1_000_000_000
```

Comment:
Why do we need this extra limit?

Reply:
There is default max set in the geth host, possibly. It is set in our nodes by default. I do not know if there is a way to test for it or if it is present for providers or is the same max. https://github.com/ethereum/go-ethereum/blob/dbd6c1324dda8bb93c7a8297446fce7998cab4e1/internal/ethapi/api.go#L2084-L2097

We dealt with this with the internal node here https://github.com/decred/dcrdex/pull/1490

Reply:
I'll add more to the comment anyway.

Reply:
The only api that I can find that gives some info about the host is `admin.nodeInfo` and we probably cannot access admin and it doesn't even have the value. If anyone knows how to sniff this value out would be helpful. There is also another config variable the rpc gas cap we really should know, but the default is 50 million so its probably ok for now: https://github.com/ethereum/go-ethereum/blob/dbd6c1324dda8bb93c7a8297446fce7998cab4e1/internal/ethapi/api.go#L1050

Reply:
I guess all the compliant providers use geth?

Reply:
Updated the comment https://github.com/decred/dcrdex/compare/0a64ba6c49a253564674ae6f973be45190e4a8e6..f004787d95fc2874562e8c873571bcbd5f6c7721

Reply:
To restate the reason:

`--rpc.txfeecap value                Sets a cap on transaction fee (in ether) that can be sent via the RPC APIs (0 = no cap) (default: 1)`

And it's probably safe to assume that at least some providers will be using geth with the default.